### PR TITLE
update several 'CREATE' related statement

### PR DESCRIPTION
**Pull request by tangenta**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs-cn#3293.

### CODE REVIEW

File: sql-statements/sql-statement-create-table-like.md [hunk 1]
```diff
@@ -52,6 +48,11 @@ mysql> SELECT * FROM t2;
 Empty set (0.00 sec)
 ```
 
+## Presplit region
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table-like.md:51]:
```suggestion
## Pre-split Region
```

File: sql-statements/sql-statement-create-table-like.md [hunk 2]
```diff
@@ -52,6 +48,11 @@ mysql> SELECT * FROM t2;
...
+
+The table built with `CREATE TABLE LIKE` inherits the attribute `PRE_SPLIT_REGIONS`. When creating a table, the region on the new table will be split.
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table-like.md:53]:
```suggestion
If the table to be copied is defined with the `PRE_SPLIT_REGIONS` attribute, the table created using the `CREATE TABLE LIKE` statement inherits this `PRE_SPLIT_REGIONS` attribute, and the Region on the new table will be split. For details of `PRE_SPLIT_REGIONS`, see [`CREATE TABLE` Statement](/sql-statements/sql-statement-create-table.md)。
```

File: sql-statements/sql-statement-create-table-like.md [hunk 3]
```diff
@@ -52,6 +48,11 @@ mysql> SELECT * FROM t2;
 Empty set (0.00 sec)
 ```
 
+## Presplit region
+
+The table built with `CREATE TABLE LIKE` inherits the attribute `PRE_SPLIT_REGIONS`. When creating a table, the region on the new table will be split. 
+如果被复制的表定义了 `PRE_SPLIT_REGIONS` 属性，则通过 `CREATE TABLE LIKE` 语句复制的表，会继承该属性并在建表时预切分 Region。For details of `PRE_SPLIT_REGIONS`, see [`CREATE TABLE` Statement](/sql-statements/sql-statement-create-table.md)。
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table-like.md:54]: Please remove this Chinese passage.

File: sql-statements/sql-statement-create-table.md [hunk 4]
```diff
@@ -215,10 +231,14 @@ The `table_option` currently only supports `AUTO_INCREMENT`, `SHARD_ROW_ID_BITS`
 | `AUTO_INCREMENT` | The initial value of the increment field | `AUTO_INCREMENT` = 5 |
 |`SHARD_ROW_ID_BITS`| To set the number of bits for the implicit `_tidb_rowid` shards |`SHARD_ROW_ID_BITS` = 4|
 |`PRE_SPLIT_REGIONS`| To pre-split `2^(PRE_SPLIT_REGIONS)` Regions when creating a table |`PRE_SPLIT_REGIONS` = 4|
+|`AUTO_ID_CACHE`| To set the auto ID cache size in a TiDB instance. By default, TiDB automatically resize it according to allocation speed |`AUTO_ID_CACHE` = 200|
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table.md:234]:
```suggestion
|`AUTO_ID_CACHE`| To set the `AUTO_ID` cache size in a TiDB instance. By default, TiDB automatically changes this size according to the allocation speed of `AUTO_ID` |`AUTO_ID_CACHE` = 200|
```

File: sql-statements/sql-statement-create-table.md [hunk 5]
```diff
@@ -215,10 +231,14 @@ The `table_option` currently only supports `AUTO_INCREMENT`, `SHARD_ROW_ID_BITS`
...
+|`AUTO_RANDOM_BASE`| To set the initial incremental part value of auto_random. This can be consider as an internal interface. For users, please ignore |`AUTO_RANDOM_BASE` = 0|
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table.md:235]:
```suggestion
|`AUTO_RANDOM_BASE`| To set the initial auto-increment part value of the `auto_random` attribute. This parameter can be considered as a part of the internal interface. Users can ignore this parameter. 
 |`AUTO_RANDOM_BASE` = 0|
```

File: sql-statements/sql-statement-create-table.md [hunk 6]
```diff
@@ -251,14 +271,15 @@ mysql> SELECT * FROM t1;
 
 ## MySQL compatibility
 
-* TiDB does not support the syntax `CREATE TEMPORARY TABLE`.
+* TiDB does not support the temporary table, but it ignores the syntax `CREATE TEMPORARY TABLE`.
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table.md:274]:
```suggestion
* TiDB does not support temporary tables, but it ignores the `CREATE TEMPORARY TABLE` syntax.
```

File: sql-statements/sql-statement-create-table.md [hunk 7]
```diff
@@ -251,14 +271,15 @@ mysql> SELECT * FROM t1;
...
 * All of the data types except spatial types are supported.
 * `FULLTEXT`, `HASH` and `SPATIAL` indexes are not supported.
 * The `KEY_BLOCK_SIZE` and `ENGINE` attributes are parsed but ignored.
-* The `index_col_name` attribute supports the length option with a maximum length limit of 3072 bytes. The length limit does not change depending on the storage engine, and character set used when building the table.
+* The `index_col_name` attribute supports the length option with a maximum length limit of 3072 bytes by default. The length limit can be changed in the `max-index-length` configuration option. For details, see [TiDB configuration file](/tidb-configuration-file.md#max-index-length).
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table.md:278]:
```suggestion
* The `index_col_name` attribute supports the length option with a maximum length limit of 3072 bytes by default. The length limit can be changed through the `max-index-length` configuration option. For details, see [TiDB configuration file](/tidb-configuration-file.md#max-index-length).
```

File: sql-statements/sql-statement-create-table.md (hunk 4 above)
Comment (tangenta) [reply sql-statements/sql-statement-create-table.md:234]: `AUTO_ID` is not a keyword. "auto ID" means "ID allocated automatically", including `auto_increment` ID and `auto_random` ID.

File: sql-statements/sql-statement-create-table.md (hunk 5 above)
Comment (tangenta) [reply sql-statements/sql-statement-create-table.md:235]: I think "auto-increment" here and the keyword "auto_increment" can confuse users.

Comment (TomShawn) [reply sql-statements/sql-statement-create-table.md]: Comment updated, PTAL.

File: sql-statements/sql-statement-create-table.md [hunk 8]
```diff
@@ -251,14 +271,15 @@ mysql> SELECT * FROM t1;
...
 * All of the data types except spatial types are supported.
 * `FULLTEXT`, `HASH` and `SPATIAL` indexes are not supported.
 * The `KEY_BLOCK_SIZE` and `ENGINE` attributes are parsed but ignored.
-* The `index_col_name` attribute supports the length option with a maximum length limit of 3072 bytes. The length limit does not change depending on the storage engine, and character set used when building the table.
+* The `index_col_name` attribute supports the length option with a maximum length limit of 3072 bytes by default. The length limit can be changed through the `max-index-length` configuration option. For details, see [TiDB configuration file](/tidb-configuration-file.md#max-index-length).
 * The `index_col_name` attribute supports the index sorting options of `ASC` and `DESC`
 * The `COMMENT` attribute supports a maximum of 1024 characters and does not support the `WITH PARSER` option.
 * TiDB supports at most 512 columns in a single table. The corresponding number limit in InnoDB is 1017, and the hard limit in MySQL is 4096.
+* For partitioned tables, only Range, Hash and Range Columns(single column) are supported. For details, see [partitoned table](/partitioned-table.md).
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-table.md:282]:
```suggestion
* For partitioned tables, only Range, Hash and Range Columns (single column) are supported. For details, see [partitoned table](/partitioned-table.md).
```

**Comment by TomShawn** [comment]
LGTM
@bb7133 PTAL

**Pull request merged**